testing.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+FILE=output.txt
+
+if test -f $FILE
+then
+	rm $FILE
+fi
+
+for unit1 in units/* 
+do
+	for unit2 in units/* 
+  do
+	if [ $unit1 != $unit2 ]
+    then
+			./a.out $unit1 $unit2 >> $FILE
+		fi
+	done
+done
```

Comment:
Ez nem az a script ami a feladat volt. Nincs összehasonlítás az elvárt kimenethez képest.

Reply:
(Ezt pótoljátok.)

Reply:
Az összehasonlítás a github action yml fájljában történik. Ahogy az az órai anyag videójában is volt.